workflow/controller/operator.go

```diff
@@ -1880,23 +1897,24 @@ func (woc *wfOperationCtx) initializeNodeOrMarkError(node *wfv1.NodeStatus, node
 }
 
 // Creates a node status that's completely initialized and marked as finished
```

Comment:
Wrong comment

Reply:
Good catch, changed the function but not the comment